api/tests/core/bookings/test_api.py

```diff
@@ -535,6 +536,69 @@ def test_book_manual_offer(self):
             assert booking.token
             assert len(booking.externalBookings) == 0
 
+        @override_features(ENABLE_CDS_IMPLEMENTATION=True, DISABLE_CDS_EXTERNAL_BOOKINGS=True)
```

Comment:
```suggestion
        @override_features(DISABLE_CDS_EXTERNAL_BOOKINGS=True)
```
`ENABLE_CDS_IMPLEMENTATION` est activé par défaut.